.github/workflows/set-version.yaml

```diff
@@ -26,15 +26,15 @@ jobs:
         sed -i -e 's/const versionMajorMinor = ".*"/const versionMajorMinor = "${{ github.event.client_payload.core_major_minor }}"/g' tests/baselines/reference/api/typescript.d.ts
         sed -i -e 's/const versionMajorMinor = ".*"/const versionMajorMinor = "${{ github.event.client_payload.core_major_minor }}"/g' tests/baselines/reference/api/tsserverlibrary.d.ts
         sed -i -e 's/const version\(: string\)\{0,1\} = .*;/const version = "${{ github.event.client_payload.package_version }}" as string;/g' src/compiler/corePublic.ts
-        npm ci
+        npm install
         npx hereby LKG
         npm test
         git diff
-        git add package.json
+        git add package.json package-lock.json
         git add src/compiler/corePublic.ts
         git add tests/baselines/reference/api/typescript.d.ts
         git add tests/baselines/reference/api/tsserverlibrary.d.ts
-        git add ./lib
+        git add --force ./lib

```

Comment:
I'm a little worried that this will work the first time, then be annoying when pulling in main another time because we'll need to force it to be included. If we use the "update-lkg" workflow to do it, then it probably will be fine, but if being done manually, it might get weird.

Maybe I should just make it remove the ignore from `.gitignore`?